documentation/5_Organizational_Agreements.md

```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
+data sharing initiative and preparing for interoperability between
+multiple data space instances or data sharing initiatives (inter) to
+pave the way for the federation of interoperable data spaces - as
+pursued by the European data strategy.
+
+The [Data Governance Act](https://digital-strategy.ec.europa.eu/en/policies/data-governance-act) also recommends both an intra data space
+(domain) governance authority and an inter data space (central)
+governance authority.
+
+'The recently proposed [Data Governance Act](https://digital-strategy.ec.europa.eu/en/policies/data-governance-act) confirms the notion of
+a governance structure constituted by multiple entities. For European
+data spaces, it is recommended to have a (domain) governance authority
+for each data space and a central governance authority overseeing all
+aspects in connection with interoperability of data spaces, i.e., the
+de-facto 'soft infrastructure'. This central authority will interact
+with all data space specific authorities.'
+
+As noted above, IDSA and the main IDS-stakeholders play an important
+role by working together to ensure the development and deployment of
+data space instances in the broader context of introducing new or
+migrating/ developing existing data sharing initiatives, (intra data
+space interoperability), and embedding them in the European ambition of
+a federation of interoperable data spaces (inter data space
+interoperability).
+
+![Relationship of OpenDEI Building
+Blocks and data space instances](media/media/image18.png)
+
+This structure of a data space reflects the role of the IDS data space
+framework in relation to the broadly used data sharing agreement
+framework that is emerging as the cloud processing framework, as shown
+on the right in Figure 1:
+
+An approach to systematically address the interoperability challenges is
+provided by the [new European interoperability framework](https://ec.europa.eu/isa2/eif_en/) developed by the
+European Commission and shown in the figure below.
+
+![Layered functional model as aligned
+with the New European Interoperability Framework ](media/EuropeanInteroperabilityFramework.png)
+
+As shown in the figure above, the framework distinguishes four functional levels
+under an overarching integrated governance approach:
+
+- *Technical level,* to provide software and hardware components for
+    controlled, sovereign and secure sharing of data.
+
+- *Semantic level,* to ensure that format and meaning of shared data
+    is preserved and understood.
+
+- *Organizational level,* to let stakeholders align goals,
+    expectations, responsibilities, and business processes.
+
+- *Legal level,* to make sure that organizations under different legal
+    jurisdictions and frameworks can share data with common legally
+    binding conditions.
+
+Governance on the identified topics distinguishes between two
+development lines for data space instances:
+
+- *Intra data space interoperability,* between the data space
```

Comment:
missing participants as the core element of interoperability in intra & inter dataspace interop